src/Pages/GradeSelect/index.js

```diff
@@ -1,7 +1,314 @@
+    setScrollTop(scrollTop);
+  }
+
+  function toggleDropdown() {
+    setDropdownActive(!isDropdownActive);
   }
+
+  function toGoNextStage(e) {
+    e.preventDefault();
+    props.history.push('/gradeselect/arrival');
+  }
+
+  function handleSelected(target) {
+    const newSelectedDep = flightLists.filter((el) => el.id === target.id);
+    setSelectedDep(...newSelectedDep);
+  }
+
+  const price = +selectedDep.price;
+
+  return (
+    <div>
+      {console.log(selectedDep)}
+      <GradeSelectMain>
+        <Wrap>
+          <SearchWidget scrollTop={scrollTop} />
+          {/* num navi */}
+          <NumNavi>
+            <NaviLists>
+              <NaviList>
+                <NaviLink to="/main">
+                  <Numbering>1</Numbering>
+                  <NaviTitle>검색</NaviTitle>
+                </NaviLink>
+              </NaviList>
+              <NaviList>
+                <NaviLink to="/gradeselect">
+                  <Numbering className="active">2</Numbering>
+                  <NaviTitle className="focus">항공편</NaviTitle>
+                </NaviLink>
+              </NaviList>
+              <NaviList>
+                <NaviLink to="orderform">
+                  <Numbering className="undone">3</Numbering>
+                  <NaviTitle className="undone">결제</NaviTitle>
+                </NaviLink>
+              </NaviList>
+            </NaviLists>
+          </NumNavi>
+          {/* title*/}
+          <Title>
+            <TitleWrap>
+              <TitleHead> 가는 편 </TitleHead>
+              <p>
+                <TitlePara>GMP 서울/김포</TitlePara>
+                <TitlePara>→</TitlePara>
+                <TitlePara>CJU 제주</TitlePara>
+              </p>
+            </TitleWrap>
+          </Title>
+
+          <form>
+            <FlightList>
+              <FlightListWrap>
+                <FilterSelector
+                  toggleDropdown={toggleDropdown}
+                  isDropdownActive={isDropdownActive}
+                />
+                <ul>
+                  {!!flightLists.length &&
+                    flightLists.map((flightList) => {
+                      return (
+                        <FlightListCard
+                          key={flightList.id}
+                          list={flightList}
+                          handleSelected={handleSelected}
+                        />
+                      );
+                    })}
+                </ul>
+              </FlightListWrap>
+            </FlightList>
+            <SubBtnBar>
+              <SubBtnBarWrap>
+                <SubBtnBarTitle>예상 결제 금액</SubBtnBarTitle>
+                <SubBtnBarFare>
+                  <TotalFare>
+                    <FareNum>
+                      {!selectedDep.price ? 0 : price.toLocaleString()}
+                    </FareNum>
+                    원
+                  </TotalFare>
+                </SubBtnBarFare>
+                <PaymentBtn
+                  className={!!selectedDep.id && 'active'}
```

Comment:
```suggestion
                  isActive={!!selectedDep.id}
```
- className에 &&를 사용하게 되면 클래스명으로 문자열 null이 들어가기 때문에 삼항연산자 사용하는게 좋습니다
- 그리고 스타일드 컴포넌트를 쓰는 만큼 isActive라는 프로퍼티를 추가해주는게 더 자연스러워보여요.

Reply:
수정하였습니다!